client/src/js/samples/components/Item.js

```diff
@@ -90,30 +103,21 @@ class SampleEntry extends React.Component {
                 <div className="sample-checkbox-overlay" onClick={this.handleCheck} />
                 {analyzeIcon}
                 <LinkContainer to={`/samples/${this.props.id}`}>
-                    <ListGroupItem className="spaced">
+                    <SpacedBox>
                         <Flex alignItems="center" style={{ userSelect: "none" }}>
-                            <FlexItem grow={1}>
```

Comment:
This can be combined to one component using `styled-components` and `LinkBox`. Remove `LinkContainer`.

See https://developer.mozilla.org/en-US/docs/Web/CSS/user-select also.